internal/config/config.go

```diff
@@ -231,3 +260,31 @@ func newCloudConnectorsConfig() CloudConnectorsConfig {
 		ResourceID:    os.Getenv(CloudResourceIDEnvVar),
 	}
 }
+
+const (
+	CloudbeatGCPListAssetPageSizeEnvVar = "CLOUDBEAT_GCP_LIST_ASSETS_PAGE_SIZE"
+	CloudbeatGCPListAssetTimeoutEnvVar  = "CLOUDBEAT_GCP_LIST_ASSETS_TIMEOUT"
+)
+
+func overwritesFromEnvVars(c *Config) {
+	log := clog.NewLogger("config")
+	logErr := func(name string, value string, err error) {
+		log.Errorf("error trying to parse variable %s with value %s: %s", name, value, err.Error())
+	}
+
```

Comment:
I think it worth to keep the message predicted and the values as keys and values to make later search in o11y easier, wdyt?

Reply:
In general, I agree, I am all about structured logging. I just tried to follow the cloudbeat's code way, where we don't use structured logging almost anywhere, but yes, we can change those two.